src/Components/Header/index.jsx

```diff
@@ -16,6 +17,7 @@ import Logo from '../../Assets/Logo.png';
 export default function Header() {
   const [style, setStyle] = useState({ display: 'none' });
   const [open, setOpen] = useState(false);
+  const dispatch = useDispatch();
 
   const showMenu = () => {
     setOpen(!open);
```

Comment:
Pay attention here, you wrote a toggeling function and not a showingMenu function. Because if I click twice, the open state will become false